source/slang/slang-ir-insts.h

```diff
@@ -4433,6 +4433,8 @@ struct IRBuilder
 
     IRGlobalConstant* emitGlobalConstant(IRType* type, IRInst* val);
 
+    IRConstant* emitPtrLit(IRType* type, void* ptr);
```

Comment:
Instead of adding this, we should change `IRBuilder::_getPtrValue` to take a type. We should bottleneck all constant inst creation through `findOrEmitConstant` as in `_getPtrValue`.